packages/stryker/src/reporters/ProgressReporter.ts

```diff
@@ -22,18 +22,21 @@ export default class ProgressBarReporter extends ProgressKeeper {
   public onMutantTested(result: MutantResult): void {
     const ticksBefore = this.progress.tested;
     super.onMutantTested(result);
+
+    const progressBarContent = Object.assign(this.progress, { etc: this.getEtc() });
```

Comment:
You could also use the spread operator here: `const progressBarContent = { ...this.progress, etc: this.getEtc() };`